invisible_cities/cities/components.py

```diff
@@ -241,6 +248,19 @@ def get_event_info(h5in):
     return h5in.root.Run.events
 
 
+def get_number_of_active_pmts(detector_db, run_number):
+    datapmt = load_db.DataPMT(detector_db, run_number)
+    return np.count_nonzero(datapmt.Active.values.astype(bool))
+
+
+def check_nonempty_indices(s1_indices, s2_indices):
+    return s1_indices.size and s2_indices.size
+
+
+def check_empty_pmap(pmap):
+    return pmap.s1s + pmap.s2s
```

Comment:
This function is a bit weird, would it not be more self consistent and more obvious for the name to return `len(pmap.s1s + pmap.s2) != 0` ?

Reply:
Not using Python's generalized booleans (empty containers being false, in this case) and, instead, fiddling around with `len` and `0`, is an anti-pattern, so please don't do that.

If you want to be explicit about the boolean aspect, just say `bool(pmap.s1s + pmap.s2)`.

But concatenating `s1s1` and `s2` just to check that they are both empty, is a bit of a waste of CPU cycles. How about `bool(pmap.s1s) or bool(pmap.s2)`?

Reply:
Better, yes, the concatenation of two lists just doesn't seem like a 'check' to me, it reads as a concatenation.

I'm happy with `return pmap.s1s or pmap.s2s` without the explicit boolean as it already looks like we're checking something

Reply:
I have just pushed @jacg proposal, but if you prefer `return pmap.s1s or pmap.s2s`, we can do that as well.

Reply:
No, that's great how it is